test/dlc_test.json

```diff
@@ -0,0 +1,329 @@
+[ {
+  "testInputs" : {
+    "params" : {
+      "oracleInfo" : {
+        "publicKey" : "a78bf5ed2841a93d4dcc1b719ebbcc90c7648746e5bd55fe7806494a93167c39",
+        "nonce" : "d2d6cb7049ef4edc2d98aa3e235e8d542573cf0c9a735516b9936aba52ae693e"
+      },
+      "contractInfo" : {
+        "8873e27fd5804a500792d3b197ec83e3d13311e161c431b223657b64ec78e355" : 0,
```

Comment:
This is not really practical to parse. I can see in your code you use a Map, but usually maps don't guarantee ordering especially across implementations so I think it would be better to use an array here.